src/moclojer/external_body/xlsx.clj

```diff
@@ -0,0 +1,23 @@
+(ns moclojer.external-body.xlsx
+  (:require [dk.ative.docjure.spreadsheet :as sheet]))
+
+(defn header-keywordize
```

Comment:
this I think do the samething just a suggestion:

```(defn header-keywordize
  "Converts header to keywords."
  [[header-row & rows]]
  (let [header-keys (map keyword header-row)]
    (map (fn [row] (zipmap header-keys row)) rows)))

(def t  [["name" "lang"]
         ["avelino" "clj, go"]
         ["chico" "clj, clj"]])

(header-keywordize t)```

({:name "avelino", :lang "clj, go"} {:name "chico", :lang "clj, clj"})


can you add some unit test for this function as well